src/components/HeroCopyElement/index.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useEffect } from "react";
+import Typography from "root/components/Typography";
+
+import "./index.css";
+
+function HeroCopyElement() {
+  const [seconds, setSeconds] = useState(0);
+  const [position, setPosition] = useState(0);
+  const [isActive, setIsActive] = useState(true);
+  const textPlaceholder = ["For founders", "For companies", "For everyone"];
+
+  function getClassName(place) {
+    if (place === position) {
+      return "line-klein-blue";
+    }
+
+    return "line-baby-blue";
+  }
+
+  function reset(value) {
+    setSeconds(value);
+    setPosition(value);
+    setIsActive(false);
+    setInterval(() => {
+      setIsActive(true);
+    }, 5000);
+  }
+
+  useEffect(() => {
+    let interval = null;
+
+    if (isActive) {
+      interval = setInterval(() => {
+        setSeconds(secs => secs + 1);
+        setPosition((seconds + 1) % 3);
+      }, 5000);
+    } else if (!isActive && seconds !== 0) {
+      clearInterval(interval);
+    }
+
+    return () => clearInterval(interval);
+  }, [isActive, seconds]);
+
+  return (
+    <div styleName="copy-section">
+      <div styleName="lines">
```

Comment:
I think we need to have a higher clickable area for the line

Reply:
What do you suggest?

Reply:
have a white background area behind the line

Reply:
Check my latest commit, I think it solved it... But instead, I used 20px

Reply:
It's ok now